frontend/src/components/StatusPage/StatusPageDetails.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Document } from "solo-types";
+import StatusPageDetailData from "./StatusPageDetailData";
+
+interface StatusPageDetailsProps {
+  document: Document;
+}
+
+const StatusPageDetails: React.FC<StatusPageDetailsProps> = ({ document }) => {
+  const { status, part, address } = document;
+
+  return (
+    <>
+      <StatusPageDetailData address={address} part={part} status={status} />
+    </>
+  );
+};
+
+export default StatusPageDetails;
```

Comment:
Why make this a separate component? OR why separate out the data component?

Reply:
This is where the timeline component will go